README.md

```diff
@@ -1,89 +1,115 @@
 
-## Install
+
+## :rocket: Install
 
 Get it from the [Google Play Store](https://play.google.com/store/apps/details?id=io.cozy.files_client),
-or head over to the [Releases page](https://github.com/cozy/cozy-mobile/releases)
-or compile it.
+or on [Aptoide Store](https://cozy.store.aptoide.com/app/market/io.cozy.files_client/103058/19682485/Cozy)
+or head over to the [Releases page](https://github.com/cozy/cozy-mobile/releases).
+
+
+## :boat: Compile yourself
+
+### :wrench: Requirement
+
+- Node v4.4.7
+- Android SDK 23.0.3 to deploy on android
+- Xcode 7.3.1 to deploy on ios
+
+Necessary path:
+
+    export ANDROID_HOME="/path/to/android-sdk-linux"
+    export PATH="$PATH:$ANDROID_HOME/tools"
+    export PATH="$PATH:$ANDROID_HOME/build-tools/23.0.3"
+    export PATH="$PATH:$ANDROID_HOME/platform-tools"
+
+
+### :package: Compile from source
 
     git clone https://github.com/cozy/cozy-mobile
-    cd cozy-mobile
-    npm install -g coffee-script
-    cake install
-
-### Resource for cordova installation
-- [ubuntu todolist](http://askubuntu.com/questions/318246/complete-installation-guide-for-android-sdk-adt-bundle-on-ubuntu)
-- [cordova manual](https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html)
-- Necessary path
-```bash
-export ANDROID_HOME="/path/to/android-sdk-linux"
-export PATH="$PATH:$ANDROID_HOME/tools"
-export PATH="$PATH:$ANDROID_HOME/build-tools/23.0.2"
-export PATH="$PATH:$ANDROID_HOME/platform-tools"
-```
-
-## Build with
-```bash
-.$ cd src
-./src$ npm run build # build coffee into js in src
-./src$ npm run buildapk # build the apk using cordova
-./src$ npm run release # sign the apk, need ./keys
-```
-
-## Hack
-
-Make your changes in src, use brunch to compile in wwww
-
-    cd src
-    npm run start
-
-To run on device / emulator
-
-    ./src/node_modules/.bin/cordova run android
-
-To run in browser,
-- start chrome with --disable-web-security --allow-file-access-from-files
-- open www/index.html
-- in browser's console run
-```
-window.isBrowserDebugging = true
-document.dispatchEvent(new Event('deviceready'));
-```
-
-Expect all things related to binary, contact or calendar to fail in browser.
-
-If you want to test on your Android device directly, please install the Android
-SDK http://developer.android.com/sdk/index.html. Then enable USB debugging and
-`cordova run android` will run the application within your phone instead of from
-the emulator.
-
-## Contribute with Transifex
+    cd cozy-mobile/src
+    npm install
+    npm run build
+
+or if you develop :loop::
+
+    npm run watch
+
+
+### :helicopter: Deploy
+
+After you can deploy with one of this command (always from `src` folder):
+
+    npm run android
+    npm run android-emulator
+    npm run ios
+    npm run ios-emulator
+
+
+### :warning: Android bug
+
+Some time we need to mention that there is a case we on testing on real device
+there can be this error ``Android Cannot access org.apache.http.client.HttpClient``
```

Comment:
You need to be aware that, some times, there is a special case when testing on real device,
where the error ``Android Cannot access org.apache.http.client.HttpClient`` can happen.